src/fetch.ts

```diff
@@ -44,6 +44,18 @@ async function downloadChromeFile(url: string) {
     }
 }
 
+async function addCustomSchemas() {
```

Comment:
Instead of duplicating the schema file, have you tried just reading the "schema-custom" files within readAllSchemaFiles?

Reply:
I have updated the pull request.